compiler/noirc_evaluator/src/errors.rs

```diff
@@ -32,8 +32,13 @@ pub enum RuntimeError {
     TypeConversion { from: String, into: String, call_stack: CallStack },
     #[error("{name:?} is not initialized")]
     UnInitialized { name: String, call_stack: CallStack },
-    #[error("Integer sized {num_bits:?} is over the max supported size of {max_num_bits:?}")]
-    UnsupportedIntegerSize { num_bits: u32, max_num_bits: u32, call_stack: CallStack },
+    #[error("Integer{} sized {num_bits:?} is over the max supported size of {max_num_bits:?}", opt_value.as_ref().map(|x| format!(" ({})", x)).unwrap_or("".to_string()))]
```

Comment:
I think it would be cleaner to not have this `opt_value` and just a separate error variant. As the error is now, for cases of `None` we are always going to have a comma trailing the error message even if there is not `opt_value`. We can have a separate error variant `InvalidBlackBoxInputBitSize` that says the wrong sized value was used for black box function input.